docs/notes.md

```diff
@@ -0,0 +1,27 @@
+# Tracking the impact of changes
+
+Some changes in your organization, either internal or external to the environment, can impact your software delivery process. For example, changes in the team structure or membership, teams adopting new working processes, or major events affecting the system stability, might have a positive or negative impact on your teams' performance.
+
+Pulse enables you to track and correlate the impact that specific changes might have on your teams' performance using **notes** to signal actions or events that took place on a specific date.
+
+![Pulse notes](images/notes.png)
+
+Pulse shows your organization notes across the different dashboards - [Accelerate overview](metrics/accelerate.md), [Changes & Reviews](metrics/accelerate-changes.md), and [Lead & Cycle time](metrics/lead-cycle-time.md). This way you can compare each performance metric before and after a specific change.
+
+!!! note
+    -   Notes are global to the organization. Any member can add or delete any note.
+    -   Pulse dashboards show the data for the maximum one year period, thus you won't be able to see notes older than that period.
```

Comment:
Edit to simplify/clarify the sentence:

```suggestion
    -   Pulse dashboards show at most one year of data, thus you won't be able to see notes older than that period.
```

Reply:
I had to rewrite the sentence to include another limitation: 
"Pulse dashboards show the latest 100 notes for the selected time range."

As I'm now referring to the "selected time range" (maximum one year, in Pulse), I believe I can skip the part about not seeing the data older than that period. Do you agree, @prcr ?

Reply:
I like how it essentially says the same thing in a simpler way. :+1:

Like this, we don't explicitly say that it will be impossible (at least for now) to see notes older than one year, but we can adjust later if we see that users have questions about the issue.